docs/application/native/guides/internationalization/i18n.md

```diff
@@ -844,17 +844,17 @@ argStyle = "short" | "medium" | "long" | "full" | "integer" | "currency" | "perc
 **Examples:**
 
 ```
-one{dog} other{dogs}                        // English plural format for dog
-one{pes} two{psa} few{psi} other{psov}      //Slovenian translation of dog in plural format
+one{dog} other{dogs}                 // English plural format for dog
+one{pies} few{psy} other{psów}       //Polish translation of dog in plural format
 ```
 
 Pattern_White_Space between syntax elements is ignored, except between the {curly braces}
 and their sub-message, and between the '=' and the number of an explicitValue.
 
-There are 6 predefined casekeyword in CLDR/ICU - 'zero', 'one', 'two', 'few', 'many' and 'other'.
-You always have to define a message text for the default plural case other
-which is contained in every rule set. If you do not specify a message text for a particular plural case,
-the message text of the plural case other gets assigned to this plural case.
+In ICU there are 6 predefined keywords: 'zero', 'one', 'two', 'few', 'many' and 'other'.
+It is obligatory to define message text for the 'other' keyword which is contained in
+every rule set. In case that you do not define specified message for a particular number object,
+the 'other' keyword message will be returned to this plural case.
 
```

Comment:
Please change as:
In International Components for Unicode (ICU) there are six predefined keywords: `zero`, `one`, `two`, `few`, `many` and `other`. It is obligatory to define message text for the `other` keyword, which is included in every rule set. In case if you do not define the specified message for a particular number object, the `other` keyword message will be returned to this plural case.